cls/SourceControl/Git/Settings.cls

```diff
@@ -32,6 +32,15 @@ Property gitUserEmail As %String(MAXLEN = 255) [ InitialExpression = {##class(So
 /// Attribution: Whether mapped items should be read-only, preventing them from being added to source control
 Property mappedItemsReadOnly As %Boolean [ InitialExpression = {##class(SourceControl.Git.Utils).MappedItemsReadOnly()} ];
 
+/// Attribution: Whether basic mode should be enabled for user ${username}, greatly simplifying the functionality of the package, requiring no knowledge of git
+Property basicMode As %String [ InitialExpression = {##class(SourceControl.Git.Utils).BasicMode()} ];
+
+/// This does not map to a setting on the screen but is used to select the value to be displayed for the user's basic mode setting
```

Comment:
The comment displays in the terminal prompt when running through Configure(), so this is currently somewhat confusing for the user:
```
Attribution: Whether basic mode should be enabled for user 'pbarton', greatly simplifying the functionality of the package, requiring no knowledge of git:
  0 => 0
This does not map to a setting on the screen but is used to select the value to be displayed for the user's basic mode setting:
  system => idk
The system's default mode. If true, the system defaults to basic mode:
  0 => 1
Settings saved.
```
I personally would remove the user-level setting for basic mode and just have the one system-level setting for basic mode. But I will defer to Tim if you have discussed otherwise.